recipes/coprarna/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "coprarna" %}
+{% set version = "alpha2" %}
+{% set md5 = "95abc9567e995331c6412f9b2f3ac6aa" %} 
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.zip
+  url: https://github.com/eggzilla/CopraRNA/archive/{{ version }}.zip
```

Comment:
should this be Patricks repo or Backofenlab?

Reply:
yes, this is only for testing until patrick has tagged the final 2.x release